src/common/str_map.cc

```diff
@@ -57,6 +57,9 @@ int get_json_str_map(
   return 0;
 }
 string trim(const string& str) {
+  if (str.size()) {
```

Comment:
s/size/empty/

Reply:
this seems to be cause for the make check failure

Reply:
Groan. I thought that was what @liewegas meant by the push and "I had the if condition reversed". If you change str.size() to !str.size() (or change size() for empty()) it all works. I guess he meant to change it but somehow didn't.